identity/app/controllers/EditProfileController.scala

```diff
@@ -29,7 +29,13 @@ object EmailPrefsProfilePage extends IdentityPage("/email-prefs", "Emails")
 object MembershipEditProfilePage extends IdentityPage("/membership/edit", "Membership")
 object recurringContributionPage extends IdentityPage("/contribution/recurring/edit", "Contributions")
 object DigiPackEditProfilePage extends IdentityPage("/digitalpack/edit", "Digital Pack")
-object ConsentJourneyPage extends IdentityPage("/consent", "Consent")
+
+sealed abstract class ConsentJourneyPage(id: String, journey: String) extends IdentityPage(id, "Consent") {
+  val journeyParam: String = journey
```

Comment:
@jfsoul not the most elegant way to have access to the param, but does work..

Reply:
`..., val journey: String)` should do the trick :)